### PR TITLE
Dbmanager adjust

### PR DESCRIPTION
**Pull request by dlowrey**
- Configured `dbmanager.execute()` to return a list of tuples
- Refactored `dbmanger.init_connections()`
- `initdb.py` now uses `dbmanager.init_connections()` to connect to MySQL server

### CODE REVIEW

**Review by asclines — CHANGES_REQUESTED** [review]
Line 56 of `dbmanager.py` throws an error.
You renamed the function from `init_connection` to `init_connections`, please refactor accordingly.

Also, could you include somewhere an example of  `exec_sql` as that would be mighty beneficial.

**Comment by dlowrey** [comment]
@asclines alright its all there.

**Review by asclines — APPROVED** [review]
Looks good!

**Comment by asclines** [comment]
@ASAAR want to review this?

File: dbmanager.py [hunk 1]
```diff
@@ -31,23 +40,38 @@ def exec_sql(connection, sql):
     :returns None if the connection or sql was None,
              a cursor containing the results of the statement otherwise
     :raises A MySqlError (likely due to an error in the SQL statement syntax)
+    
+    example: 
+        
+        from testdb.py
+        import dbmanager
```
Comment (isaac-gs) [inline dbmanager.py]: Maybe I'm just misunderstanding, but what's going on with these two lines? Shouldn't it be `from X import Y` in a single line?

File: dbmanager.py [hunk 2]
```diff
@@ -31,23 +40,38 @@ def exec_sql(connection, sql):
...
+        connection = dbmanager.init_connection('MySQLusername', 'MySQLpassword')
+        sql = 'SELECT * FROM locations LIMIT 10;'
+        result = dbmanager.exec_sql(connection, sql)
+        print(result)
     """
     cursor = None
     if sql and connection:
```
Comment (isaac-gs) [inline dbmanager.py]: Are sql and connection guaranteed by this point? If not, else? Otherwise, why keep the if statement?

**Review by isaac-gs — COMMENTED** [review]
Overall looks good, but I have a few questions

File: dbmanager.py (hunk 1 above)
Comment (dlowrey) [reply dbmanager.py]: @ASAAR Actually I meant 'From the python file testdb.py, do this stuff:' import dbmanager...

File: dbmanager.py (hunk 2 above)
Comment (dlowrey) [reply dbmanager.py]: @ASAAR You're right, besides if there was a problem with either it would get caught in the except block.

**Comment by dlowrey** [comment]
@asclines @ASAAR Making a few changes per @ASAAR comments then I will merge.

**Comment by dlowrey** [comment]
@ASAAR @asclines I fixed the above lines but I am not authorized to merge.

**Comment by asclines** [comment]
@dlowrey waiting for second reviewer to approve.

**Review by isaac-gs — APPROVED** [review]

**Pull request merged**